source/_docs/custom-certificates.md

```diff
@@ -0,0 +1,146 @@
+| **Delivery**                                                          | US Datacenter             | [Global CDN](/docs/global-cdn)  | [Global CDN](/docs/global-cdn)        |
+| **Encryption Endpoint**                                               | Load Balancer             | Application Container           | Application Container                 |
+
+\* Browser compatibility and SSL Labs score is guaranteed for shared Let’s Encrypt certificates. The same results are typical for a custom certificate from a mainstream CA with mainstream attributes, but not guaranteed.  For custom certificates, compatibility and SSL Labs score depends on attributes of that certificate, such as number of SAN entries, CA and signing algorithm.
+
+## Frequently Asked Questions
+
+### Do I need a separate certificate for each site in my organization?
+
+Nope! You can use the a single certificate to cover multiple domains spread across various environments or sites. This capability is enabled because the Global CDN uses a technology called Server Name Indication (SNI), which automatically matches inbound requests with an appropriate certificate, including custom certificates.
+
+### How long will it take to load my certificate into Pantheon?
+
+Please allow two business days to get a CSR and load the certificate.
+
+### What about for sites purchased online?
+
+Custom certificates are currently available to customers with a Pantheon contract using Legacy SSL. We have no current plans to offer it to Basic or Performance sites. If you pay for your site with a credit card and are interested in using a custom certificate, please [let us know](@ari-gold FEEEEED ME).
+
+### Will custom certificates be self-serve?
+
+The concierge service is designed to quickly guide you through the steps required to deliver HTTPS on the Global CDN using your custom certificate. Due to the limited demand, and infrequent need to load custom certificates, the concierge service may be the best fit, although we are evaluating following up with a self-serve option.
+
+
+### Which certificates do I submit?
+
+Include the end-client certificate for your named domains, as well as the intermediate certificate, in separate files.
+
+
+### What is the maximum number of SAN entries?
+
+For the broadest client compatibility we recommend limiting the number of Subject Alternate Names to 100.
+
+
+### Are private keys available for export?
+
+Private keys are just that, private, and not available for export. They are stored securely, server side, and it’s a security best practice to not share private keys among different deployments. If you manage multiple domains, with some on Pantheon, and some outside of Pantheon, then we recommend using separate certificates, and we are happy to provide you with a new Certificate Signing Request (CSR) so we can deploy a certificate on Pantheon that only has the domains served on Pantheon.
+
+
+### What are the Global CDN IP addresses?
```

Comment:
Suggest new FAQ to address whether the IP address is static/dedicated

Reply:
@rachelwhitton I could see that information being part of this FAQ answer.

@ari-gold am I correct in thinking that the IP address is static, but not dedicated?

Reply:
yes, isn't that info already in the tech specs table?